Tests/AK/TestMath.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <AK/Math.h>
+#include <AK/NumericLimits.h>
```

Comment:
Seems to also need

`#include <AK/IntegralMath.h>
`
... at least for `Meta/serenity.sh build lagom` on a Mac